wikimedia/wikipedia-ios
# Remove old editor from project setup / scripts

**Phabricator:**
https://phabricator.wikimedia.org/T357074

### Notes
This PR removes CodeMirror references in our setup scripts, update languages script and removes the submodule. I got a bit sidetracked digging into why ClangFormat failed for me after the test steps, so I put a fix in for that at the end. 

### Test Steps
1. Clone a fresh repo in a new directory.
2. Run the `./scripts/setup` script in the root directory.
3. Confirm project runs. 
4. Try adding a poorly-formatted method to AppViewController.m.
5. Try committing, confirm you see a "clang-format made formatting changes - verify them and re-commit.". Check git and confirm AppViewController.m was automatically formatted.
6. Undo changes from previous steps. Run Update Languages utility. Confirm it completes, and you see git changes for the language and namespaces json files. Do not commit.
